experiments/prompt-injection-boundary-tags/rounds/round3/harness/run_experiment.py

```
@@ -0,0 +1,761 @@
+
+    payloads = load_payloads(PAYLOAD_PATH)
+    payload_items = sorted(payloads.items())
+    if args.payload_limit > 0:
+        payload_items = payload_items[: args.payload_limit]
+
+    timestamp = datetime.now(timezone.utc).strftime("%Y%m%d_%H%M%S")
+    run_id = f"round3_ablation_{timestamp}"
+    output_path = args.output or (DATA_DIR / f"ablation_results_{timestamp}.csv")
+    latest_path = DATA_DIR / "ablation_results_latest.csv"
+
+    rng = random.Random(args.seed)
```

Ruff (0.14.14)

[error] Local variable `rng` is assigned to but never used

Remove assignment to unused variable `rng`

(F841)